ons-spark/raw-notebooks/checkpoint-staging/r_input.R

```diff
@@ -0,0 +1,159 @@
+options(warn = -1) 
+library(sparklyr)
+library(dplyr)
+library(DBI)
```

Comment:
One more thing, whenever a function from this library is used (so I think just dbExecute), make sure you prefix it with DBI::